4-appfactory/README.md

```diff
@@ -1,36 +1,97 @@
+This phase streamlines the process of onboarding applications to a Google Cloud environment by automating project creation, repository setup, and CI/CD pipeline configuration. This reduces manual effort, enforces consistent configurations, and accelerates application delivery.
 
 An overview of the application factory pipeline is shown below.
+
 ![Enterprise Application application factory diagram](../assets/eab-app-factory.svg)
 
 The application factory creates the following resources as defined in the [`app-group-baseline`](./modules/app-group-baseline/) submodule:
 
-* **Application admin project:** A project dedicated for application administration and management.
-* **Application environment projects:** A project for the application for each environment (e.g., development, nonproduction, production).
-* **Infrastructure repository:** A Git repository containing the Terraform configuration for the application infrastructure.
-* **Application infrastucture pipeline:** A Cloud Build pipeline for deploying the application infrastructure specified as Terraform.
+- __Application Admin Project (Optional):__ A new Google Cloud project to host the application's CI/CD pipelines and related resources. This project is created if `create_admin_project` is set to `true`.
+- __Application Infrastructure Projects (Optional):__ Environment-specific Google Cloud projects to host the application's infrastructure resources (e.g., GKE clusters, databases). These projects are created if `create_infra_project` is set to `true`.
+- __Cloud Source Repository or 2nd Gen Cloud Build Repository:__ Creates a repository for the application's infrastructure code. It defaults to Cloud Source Repository if `cloudbuildv2_repository_config` is not provided, otherwise it uses the specified 2nd Gen Cloud Build repository connection.
+- __Cloud Build Triggers:__ Configures Cloud Build triggers to automatically run Terraform plan and apply jobs on code changes in the infrastructure repository.
+- __Cloud Build Service Account:__ Creates or uses a Cloud Build service account with the necessary IAM roles to manage infrastructure resources.
+- __Cloud Storage Buckets:__ Creates Cloud Storage buckets for storing Cloud Build artifacts, logs, and Terraform state.
+- __VPC Service Controls (Optional):__ Configures VPC-SC perimeter and access levels, if `service_perimeter_name` is set.
 
 It will also create an Application Folder to group your admin projects under it, for example:
 
 ```txt
 .
-└── fldr-common/
-    ├── default-example/
-    │   ├── hello-world-admin
-    │   └── ...
-    ├── cymbal-bank/
-    │   ├── accounts-userservice-admin
-    │   ├── accounts-contacts-admin
-    │   ├── ledger-ledger-writer-admin
-    │   └── ...
+└── fldr-seed/
+  ├── fldr-common/
+  │   |── default-example/
+  │       ├── hello-world-admin
+  │       └── ...
+  │   |── cymbal-bank/
+  │       ├── accounts-userservice-admin
+  │       ├── accounts-contacts-admin
```

Comment:
since it is in `common`  is the same value is used in all envs?

Reply:
yes